Tasks/Task1/Task1.js

```diff
@@ -0,0 +1,123 @@
+﻿function Add(a, b) {
+    return a + b;
+}
+
+function Sub(a, b) {
+    return a - b;
+}
+
+function Mul(a, b) {
+    return a * b;
+}
+
+function Div(a, b) {
+    if (b != 0) {
+        return a / b;
+    } else {
+        document.write("Divide by zero!");
+    }
+}
+
+function IntegerDivision(a, b) {
+    if (b != 0) {
```

Comment:
Don't use`if-else` and other statements - you just should return result and not worrying about other situations